app/components/base/form/editable-table/editable-table.component.ts

```diff
@@ -109,6 +109,10 @@ export class EditableTableComponent implements ControlValueAccessor, Validator,
         return row;
     }
 
+    public trackIndex(index, row: any) {
```

Comment:
here actually you could try by the item as it's an option of type string. should be more efficient(Though doesn't really matter here as its small list that doesn't get updated)

Reply:
I will use trackRows which is already defined above.